Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM python:3.7-alpine
```

Comment:
Since the Dockerfile is only 16 lines I'd say not much complexity is added. It also helps for running the program without having to set up python + python packages locally (also helpful to test that you didn't forget to include newly used packages in the setup.py file). You can just build the Dockerfile and run it on any platform. In any case it is used for the deployment to https://conferatur.viaa.be at the moment as well.